### ISSUE TITLE
Conjunctival xerosis

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10939]
Preferred term label:
Conjunctival xerosis

Synonyms
Conjunctival dryness, Dry conjunctiva

Definition (free text, please give PubMed ID)
A condition characterized by dryness of the conjunctiva, often associated with keratinization of the conjunctival epithelium. Conjunctival xerosis is commonly linked to vitamin A deficiency, severe dry eye disease, or systemic conditions such as Sjögren syndrome. Clinical features may include a dull, rough conjunctival surface and reduced mucosal secretions.

Parent term (use hpo.jax.org/app)
Abnormality of the conjunctiva (HP:502)

Diseases characterized by this term
Vitamin A deficiency, Sjögren syndrome, Stevens-Johnson syndrome

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Can we provide some guidance to users as to how to differentiate this finding from 
[Keratoconjunctivitis sicca HP:0001097](https://hpo.jax.org/browse/term/HP:0001097)?
thanks!

**Comment by daboe01** [comment]
the suggested term is for Xerophthalmia
<https://eyewiki.org/Xerophthalmia>

this is a keratinisation of the conjunctiva, which never happens in keratoconjunctivitis sicca.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001193
- label: Conjunctival xerosis
- parents: [Abnormal conjunctiva morphology (HP:0000502)](https://hpo.jax.org/app/browse/term/HP:0000502)
- pmid: PMID:28939634
- def: A condition characterized by dryness of the conjunctiva, often associated with keratinization of the conjunctival epithelium. Conjunctival xerosis is commonly linked to vitamin A deficiency, severe dry eye disease, or systemic conditions such as Sjogren syndrome. Clinical features may include a dull, rough conjunctival surface and reduced mucosal secretions.
- comment: Diseases characterized by this term include vitamin A deficiency, Sjogren syndrome, and Stevens-Johnson syndrome.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10939
- synonyms: Conjunctival dryness(NONE)|Dry conjunctiva(NONE)
